capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.name.capitalize
```

Comment:
- [ ] For this, the method is supposed to capitalize the output of @nameable.correct_name. For this reason, it would be better to have it like this: ```@nameable.correct_name.capitalize```

![image](https://user-images.githubusercontent.com/86318284/199222774-da210b3d-a7b5-4753-85fa-98357715f4df.png)